generated/nimodinst/session.py

```diff
@@ -148,7 +148,7 @@ def __exit__(self, exc_type, exc_value, traceback):
     # TODO(texasaggie97) Rewrite to use session function instead of library once buffer
     #   retrieval is working
     def _get_error_description(self, error_code):
-        buffer_size = library.niModInst_GetExtendedErrorInfo(0, None)
+        buffer_size = self.library.niModInst_GetExtendedErrorInfo(0, None)
```

Comment:
This would have been prevented were we using the code-generated methods.
Opened #166